defaults/main.yml

```diff
@@ -60,3 +60,49 @@ steamengine_java_base_opts:
 steamengine_java_opts:
   - "-Xmx512M"
   - "-Xms512M"
+
+# DRUPAL
+# Path of drush launcher
+steamengine_drush_launcher_path: "/usr/local/bin/drush"
+# Command run at the end of deployment
+steamengine_drupal_drush_command_deployment:
+  - "deploy -y"
+# List of directory we need write permission
+steamengine_drupal_path_with_write_permission:
+  - "web/sites/default/files"
+# List of source template and target path for configuration to put inside the projet (setting.php.j2: web/site/default/setting.php )
+steamengine_drupal_project_setting_block: |
+  $databases['default']['default'] = array (
+    'database' => 'drupal',
+    'username' => 'drupal',
+    'password' => 'drupal',
+    'prefix' => '',
+    'host' => 'localhost',
+    'port' => '3306',
+    'namespace' => 'Drupal\\Core\\Database\\Driver\\mysql',
+    'driver' => 'mysql',
+  );
+  $settings['hash_salt'] = 'ChangeMe';
+  $config_directories['sync'] = '../config';
+# List of path that should be persisted in a folder outside the projet
+# The key is the folder persisted outside the root project, the value is the name (path) of the folder inside the project. A link will be created on it to the persisted folder
+steamengine_drupal_persistent_directory: 
+  files: "web/sites/default/files"
+# Directory for public file
+steamengine_drupal_public_directory: "web"
+# If is new install of drupal, deploy db dump
+steamengine_drupal_install_from_db_dump: false
```

Comment:
There is no need for two variables for db_dump, if steamengine_db_dump_url is set consider that it should be restored (steamengine_drupal_install_from_db_dump is true).